### PR TITLE
Remove greek-letter keyword arguments

### PR DESCRIPTION
**Pull request by mcabbott**
The interface should be ascii. And especially should never require you to type `ε` not  `ϵ` or the reverse.

### CODE REVIEW

File: test/losses.jl [hunk 1]
```diff
@@ -88,8 +89,8 @@ y_dis[1,:], y_dis[2,:] = y_dis[2,:], y_dis[1,:]
   @test crossentropy(ŷ, y_smoothed) ≈ lossvalue_smoothed
   @test crossentropy(ylp, label_smoothing(yl, 2sf)) ≈ -sum(yls.*log.(ylp))
   @test crossentropy(ylp, yl) ≈ -sum(yl.*log.(ylp))
-  @test iszero(crossentropy(y_same, ya, ϵ=0))
-  @test iszero(crossentropy(ya, ya, ϵ=0))
+  @test iszero(crossentropy(y_same, ya, ϵ=0))  # ε is deprecated
```
Comment (ToucheSir) [inline test/losses.jl:92]: What's our stance on using `@test_deprecated`? I'm surprised there appear to be zero usages of it in Flux, perhaps this repo proceeds it?

**Comment by CarloLucibello** [comment]
have there been actual complaints about non-ASCII characters?

**Comment by mcabbott** [comment]
I thought this was a fairly standard complaint. Sometime a misguided one, when people assume that e.g. `crossentropy(ŷ, y)` has a non-ascii named argument.

File: src/losses/utils.jl [hunk 2]
```diff
@@ -36,3 +36,12 @@ end
...
 ChainRulesCore.@non_differentiable _check_sizes(ŷ::Any, y::Any)
+
+# Greek-letter keywords deprecated in Flux 0.13
+# Arguments (old => new, :function, "β" => "beta")
+function _greek_ascii_depwarn(βbeta::Pair, func = :loss, names = "" => "")
+  Base.depwarn("""loss function $func no longer accepts greek-letter keyword $(names.first)
+    please use ascii $(names.second) instead""", func)
+  βbeta.first
+end
+_greek_ascii_depwarn(βbeta::Pair{Nothing}, _...) = βbeta.second
```
[the remark this answers is not in the document]
Comment (mcabbott) [reply src/losses/utils.jl:47]: It could live somewhere else, but where? And is it worth it?

(Most uses are in Losses, I spotted the normalisation ones afterwards, just 3.)

[the remark this answers is not in the document]
Comment (ToucheSir) [reply src/losses/utils.jl:47]: utils.jl? That's included before any layers or loss functions are defined.

**Review by ToucheSir — APPROVED** [review]
I think we should look into using `@test_deprecated` for most of the deprecated functionality in Flux itself, but that can be a separate PR.
